Handler/Image.hs

```diff
@@ -1,14 +1,17 @@
-{-# LANGUAGE RecordWildCards #-}
+{-# LANGUAGE CPP, RecordWildCards #-}
 
 module Handler.Image where
 
+#if MIN_VERSION_base(4,8,0)
+import Import hiding (foldMap)
```

Comment:
Hiding something that does not exist is only a warning -- can probably get by without the CPP here.  Also, isn't `Import` ours?  We could just not re-export foldMap...